tables/uz-g1.utb

```diff
@@ -50,15 +52,39 @@ uplow \x049A\x049B 13456      #қ
 uplow \x040E\x045E 1236       #ў
 
 # Equivalent to the additional braille adaptations.
-word sh 156
-word ch 12345
-word o’ 1236
-word g‘ 12456
+always sh 156
+always ch 12345
+always o’ 1236
+always o‘ 1236
+always o` 1236
+always g‘ 12456
+always g’ 12456
+always g` 12456
+
+# Symbol fixes
+math % 3456-245-356
+math ° 46-356
+punctuation ( 2356
+punctuation ) 2356
+punctuation { 126
+punctuation } 345
+noback punctuation [ 126
+noback punctuation ] 345
+sign @ 4-1
+sign \x2018 3
+sign \x0301 4 # acute accent
+
+noback pass2 @46-46-46 @46
+attribute roman IVXLCDM
 
 include ru-litbrl.ctb
 
 # Remove indicators before capital letters and between cyrillic and latin letters.
+attribute roman IVXLCDM
+
 noback pass3 @6 ?
-noback pass3 @46 ?
+noback pass3 @46-356 *#1=1
+noback pass3 [@46]!%roman ?
+noback pass3 [@46]@1236 ?
```

Comment:
These rules deserve some more explanation I think. It's not so obvious what happens here.

Reply:
All three rules handle capital letter sign:
`noback pass3 @46-356 *#1=1` -> Keeping pattern 46-356 for degree sign (°), otherwise the subsequent rule would delete 46, and 356 would remain for the sign (which is not valid).
`noback pass3 [@46]!%roman ?` -> If not a roman capital letter is encountered after capital sign, remove the pattern 46 (because they don't use capital letters, they are using it only for roman numbers).
`noback pass3 [@46]@1236 ?` -> Capital "o" or "g" with accent are not entering in the previous rule, hence we have to remove capsletter sign manually.

Should I write a more concise explanation of these in the table?

Reply:
That would be nice, thanks.